data/syndicate world jobs.txt

```diff
@@ -195,14 +220,40 @@ mission "Syndicate target practice [1]"
 		random < 20
 		"combat rating" > 50
 		"combat rating" < 200
+		"destroyed Syndicate target ship" < 1
```

Comment:
Destroy one ship and never again seems a little harsh. Accidents do happen.  Three would seem more appropriate.